### PR TITLE
Update to 1.0.1

### PR DESCRIPTION
**Pull request by JeanChristopheMorinPerso**
Upstream: https://github.com/jmespath/jmespath.py/tree/1.0.1

### CODE REVIEW

**Review by lorepirri — APPROVED** [review]
[PKG-3284] jmespath 1.0.1

**Destination channel**: `defaults`

**Dependency for**
- AnacondaRecipes/botocore-feedstock#55

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,43 +1,48 @@
...
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: b85d0567b8666149a93172712e68920734333c0ce7e89b78b3e987f71e5ed4f9
+  sha256: 90261b206d6defd58fdd5e85f478bf633a2901798906be2ad389150c5c60edbe
 
 build:
   number: 0
-  noarch: python
-  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+  skip: true  # [py<37]
+  script: "{{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed -vv"
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:15]:
```suggestion
  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed -vv
```

**Review by skupr-anaconda — APPROVED** [review]
LGTM but redundant double quotes

File: recipe/meta.yaml (hunk 1 above)
Comment (JeanChristopheMorinPerso) [reply recipe/meta.yaml:15]: That's totally valid YAML, so I won't do it.

**Pull request merged**